tests/plugins/images/test_markdown.py

```diff
@@ -8,9 +8,17 @@ def test_markdown(self):
         md_text = md.convert("[image:1 align:left]")
         self.assertIn("<figure", md_text)
         self.assertNotIn("[image:1 align:left]", md_text)
+        md_text = md.convert("image: [image:1 align:left]\nadasd")
+        self.assertIn("<figure", md_text)
+        md_text = md.convert("image: [image:1 align:right medium]\nadasd")
+        self.assertIn("<img src", md_text)
```

Comment:
I think when I tried checking for `<img` in `md_text`, it failed because the output contained "Image not found" instead. Is there another mixin class for adding a test image?